lib/matplotlib/tests/test_axes.py

```diff
@@ -3600,6 +3600,16 @@ def test_errorbar():
     ax.set_title("Simplest errorbars, 0.2 in x, 0.4 in y")
 
 
+@image_comparison(['errorbar_polar_caps'], extensions=['png'])
+def test_errorbar_polar_caps():
+    fig = plt.figure()
+    ax = plt.subplot(111, projection='polar')
+    theta = np.arange(0, 2*np.pi, np.pi / 4)
+    r = theta / np.pi / 2 + 0.5
+    ax.errorbar(theta, r, xerr=0.15, yerr=0.1)
```

Comment:
```suggestion
    ax.errorbar(theta, r, xerr=0.15, yerr=0.1, fmt='o')
```
Having a straight lines connecting the data points is rather misleading in polar plots. Even though this is just a test, let's use markers instead.